### PR TITLE
Issue #SB-25141 fix: Removed unwanted api calls

### PR DESCRIPTION
**Pull request by sudip-mukherjee**

### CODE REVIEW

File: src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts [hunk 1]
```diff
@@ -127,29 +127,20 @@ export class ActivitySearchComponent implements OnInit, OnDestroy {
...
-      .subscribe(({ params, queryParams, group }) => {
-        const user = _.find(_.get(group, 'members'), (m) => _.get(m, 'userId') === this.userService.userid);
-
-        /* istanbul ignore else */
-        if (!user || _.get(user, 'role') === 'member' || _.get(user, 'status') === 'inactive' || _.get(group, 'status') === 'inactive') {
-          this.toasterService.warning(this.resourceService.messages.emsg.noAdminRole);
-          this.groupsService.goBack();
-        } else {
-          this.groupData = this.groupsService.addGroupFields(group);
+      .subscribe(({ params, queryParams }) => {
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts:139]: Where did you handled this logic. Is it taken care by existing code

> 
>  if (!user || _.get(user, 'role') === 'member' || _.get(user, 'status') === 'inactive' || _.get(group, 'status') === 'inactive') {
>           this.toasterService.warning(this.resourceService.messages.emsg.noAdminRole);
>           this.groupsService.goBack();
>

File: src/app/client/src/app/modules/groups/components/group-members/group-members.component.ts [hunk 2]
```diff
@@ -61,10 +61,6 @@ export class GroupMembersComponent implements OnInit, OnDestroy {
           this.addTelemetry('member-card-menu-close');
         }
       });
-    this.groupsService.showLoader.subscribe(showLoader => {
-      this.showLoader = showLoader;
-    });
-
     this.groupsService.membersList.subscribe(members => {
```
Comment (vinukumar-vs) [inline src/app/client/src/app/modules/groups/components/group-members/group-members.component.ts:64]: Why do we need these sunbscribe, when ther eis not api call itself

**Review by vinukumar-vs — CHANGES_REQUESTED** [review]

File: src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts (hunk 1 above)
Comment (sudip-mukherjee) [reply src/app/client/src/app/modules/groups/components/activity/activity-search/activity-search.component.ts:139]: these scenarios are no longer applicable, currently, the user can not go inside the add activity page if 
1. he is a member
2. group is deactivated

File: src/app/client/src/app/modules/groups/components/group-members/group-members.component.ts (hunk 2 above)
Comment (sudip-mukherjee) [reply src/app/client/src/app/modules/groups/components/group-members/group-members.component.ts:64]: these subscriptions are not related to API . these are events to detect the changes. please have a look at groups.service.ts.

**Review by vinukumar-vs — APPROVED** [review]

**Pull request merged**